data/campaigns/World_Conquest/era/factions/The_Defiance.cfg

```diff
@@ -0,0 +1,31 @@
+#textdomain wesnoth-wc
+
+# Dunefolk + Undead
+
+#define MULTIPLAYER_SIDE_THE_DEFIANCE
+    [multiplayer_side]
+        id=The_Defiance
+        name=_"The Defiance"
+        recruit=Dune Rover,Skeleton Archer,Dune Burner,Dark Adept,Dune Soldier,Skeleton,Dune Rider,Ghost,Dune Skirmisher,Ghoul,Walking Corpse,Walking Corpse,Dune Herbalist,Dune Herbalist,Falcon,Vampire Bat
+        image={IMG_DEFIANCE}
+        type=random
+        leader= Dune Explorer,Dune Swordsman,Dune Spearguard,Dune Skirmisher,Dune Apothecary,Dune Scorcher,Dark Sorcerer,Bone Shooter,Revenant,Necrophage
+        random_leader= Dune Explorer,Dune Swordsman,Dune Spearguard,Dune Skirmisher,Dune Apothecary,Dune Scorcher,Dark Sorcerer,Bone Shooter,Revenant,Necrophage
+        [world_conquest_data]
+            commanders=Dunefolk,Undead
+            heroes=Dunefolk,Undead
```

Comment:
Heroes are exactly the same as factions, I would recommend changing to Dunesfolk, Northerners.

Reply:
Done